packages/Avatar/src/helpers.js

```diff
@@ -0,0 +1,27 @@
+import Tokens from "@paprika/tokens";
+
+export default function getAvatarColors(text) {
+  const COLORS = [
+    { background: Tokens.color.chartColor01, font: Tokens.color.white },
```

Comment:
nip:

const color = Tokens.color;
then:

```js
    { background: color.chartColor02, font: color.white },
    { background: color.chartColor03, font: color.black },
     ...
```

Reply:
@nahumzs I think you mean `nit` 😉

Reply:
nip the duplication in the bud!

Reply:
actually tristan gave very similar feedback and i told him "no way"

Reply:
so let's ignore this comment then.